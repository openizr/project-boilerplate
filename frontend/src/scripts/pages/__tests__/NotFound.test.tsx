/**
 * @jest-environment jsdom
 */

import * as React from 'react';
import NotFound from 'scripts/pages/NotFound';
import { render } from '@testing-library/react';

describe('pages/NotFound', () => {
  beforeEach(() => {
    jest.clearAllMocks();
  });

  test('renders correctly - basic', () => {
    const { container } = render(<NotFound locale={{ LABEL_TEST: 'Test' }} />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
