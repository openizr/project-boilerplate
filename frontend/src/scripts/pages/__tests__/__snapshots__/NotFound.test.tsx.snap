// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pages/NotFound renders correctly - basic 1`] = `
<div>
  Not Found.
  object
</div>
`;
